cmd/machine-config-daemon/pivot.go

```diff
@@ -365,14 +227,24 @@ func run(_ *cobra.Command, args []string) error {
 		changed = true
 	}
 
+	_, err = os.Stat(runPivotRebootFile)
```

Comment:
Is there a reason to check this if `!changed`?

Reply:
Or if `reboot` is set?

Reply:
Hm you're right, I regressed this compared to https://github.com/openshift/pivot/blob/63d3093d32f25649241cc09ae5fdc35181ead73e/cmd/root.go#L409

Reply:
How about when `reboot` is set by cmdline flag?